deploy/stacks/s3_resources.py

```diff
@@ -96,5 +99,13 @@ def __init__(self, scope, id, envname='dev', resource_prefix='dataall', **kwargs
             f'{resource_prefix}-{envname}-access-logs-bucket-name',
             export_name=f'{resource_prefix}-{envname}-access-logs-bucket-name',
             value=self.bucket.bucket_name,
-            description=f'{resource_prefix}-{envname}-access-logs-bucket-name',
+            description=f'DO NOT USE {resource_prefix}-{envname}-access-logs-bucket-name',
```

Comment:
There was a bucket and `access-logs-bucket-name` was pointing to the `resources-bucket`. To fix it I had to create a new CFN export and still keep the old one, otherwise pipelines would break.

Reply:
They will have 2 CFN exports and they should use the new one.

We cannot delete/update the old/wrong export because it is used in the FrontEnd deployment step and when an export is used it cannot be delete/modified.